pkgs/applications/graphics/tesseract/default.nix

```diff
@@ -73,7 +70,7 @@ let
   });
 
   tesseract = (if enableLanguages == [] then tesseractBase else tesseractWithData) // {
-    inherit languages tesseractBase;
+    inherit tesseractBase tesseractLanguages tessdata;
   };
```

Comment:
Why not just set `passthru` in `overrideAttrs`?

Reply:
Because these attrs should also be passed thru `tesseractBase`.
We could also define two identical `passthru`s in `tesseractBase` and `tesseractWithData`.
Although redundant, this might improve discoverability for users. What do you prefer?

Reply:
We could also [do this](https://github.com/erikarvstedt/nixpkgs/commit/d5422f0f8ee44d4bfa9bb6eed6e79b40316d213c).